manuscripts/report.py

```diff
@@ -134,11 +134,19 @@ def __init__(self, es_url, start, end, data_dir=None, filters=None,
             self.data_sources.remove('github')
             self.data_sources += ['github_issues', 'github_prs']
         self.data_sources = list(set(self.data_sources))
+        # Check for custom indices
+        self.indices = indices if indices else []
         # End temporal hack
         self.config = self.__get_config(self.data_sources)
         self.report_name = report_name
         self.projects = projects
 
+        # Create an index name dict which, for each data_source, will give the
+        # name of the elasticsearch index that has to be used.
+        self.index_dict = defaultdict(lambda: None)
```

Comment:
I think you can just initialize this to `{}`, I don't see a need for defaultdict. But please, let us know if you find it important for some reason.

Reply:
I used a default dict because in case a key is not present, it will return a `None` for the value instead of a `KeyError`.